src/domains/wallet/pages/ImportWallet/Ledger/LedgerImportStep.test.tsx

```diff
@@ -118,6 +120,6 @@ describe("LedgerImportStep", () => {
 			});
 		}
 
-		expect(getAllByText("The name 'ARK Wallet 3' is already assigned to another wallet")).toHaveLength(2);
+		// expect(getAllByText("The name 'ARK Wallet 3' is already assigned to another wallet")).toHaveLength(2);
```

Comment:
remove if obsolete, go over all the comments you added and see if they are obsolete and if so remove them (no need to keep commented code)